recipe/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = environ.get("PANDAS_VERSION") %}
```

Comment:
from pandas-release and deleted the current recipe in pandas (may still want to retain existing though as builds from source instead of sdist)